### ISSUE TITLE
Teldrassil: Home of the Kaldorai

### ISSUE THREAD

**Issue by Djurdjevdan** [Valhalla-Gaming/Tracker#5451]
This is final quest of Night Elf starting zone. After wisiting Moonwells you are sent to Tyrabde to deliver her water. Problem is quest item is missing. 

NOTE: I did this with my old character that is level caped. When I made him, this quest waa bugged so I returned with him now to compleat all the quests that were bugged before. It may be possible that I lost the quest item making me unable to finish the quest.

**Comment by carminejackal** [comment]
I can confirm it does work (as I just completed it last night), but you can get it out of order as if offers it along side another quest that sends you to get water. I chose to do them in order, thus I'm not sure if doing it out of order causes glitches. The solution to fixing such glitches usually is abandoning the quest and re-accepting it, since most quests of the 'take x to x person' tend to provide the item in question.

**Comment by Djurdjevdan** [comment]
You are right. After restarting 2-3 times, I got quest item. I don't know why it got buggedin a first place.
